### ISSUE TITLE
The Russian language is not shown in Gin Docs

### ISSUE THREAD

**Issue by KostLinux** [gin-gonic/website#273]
@appleboy They Russian language is not shown in [Gin Docs](https://gin-gonic.com/docs/) even tho I've added everything was needed there.

**Comment by deining** [comment]
Inside `config.toml`, have a look at the first line of the section for Russian language, it declares the section as Portuguese:

https://github.com/gin-gonic/website/blob/a6b5d1fb116ff092c05f082c7e0a7a9b11555368/config.toml#L133-L139

Fix this copy and paste error and  change line 133 from `[languages.pt]` to `[languages.ru]` and see whether it works then.

**Comment by deining** [comment]
See my PR #274 that makes the website work again.

**Comment by KostLinux** [comment]
@deining there's still no russian lang on page even though the PR got merged. ![image](https://github.com/gin-gonic/website/assets/46846027/a8f82725-8f26-46b3-a1bc-496213ce9703)

**Comment by deining** [comment]
> @deining there's still no russian lang on page even though the PR got merged.

Yes, you are right. **But:** the current web site doesn't represent the HEAD of repo, the site simply wasn't deployed again after my PR was merged. Once the site will be redeployed by the maintainers, Russian language will show up 😄.

**Comment by deining** [comment]
Started with #276 to fix the deployment, but this site hasn't seen care for a long time! Will continue later!

**Comment by deining** [comment]
Finished my work on #276, once this PR got merged, Russian language should be selectable!

**Comment by KostLinux** [comment]
@appleboy @deining i think we should revert the last merge, cause now the language selector is not working :D

**Comment by luigimorel** [comment]
@KostLinux we can wait for @appleboy to merge #280. I use the example Docsy site from Google which fixes most of the broken functionality issues in the website.

**Comment by luigimorel** [comment]
@KostLinux  @appleboy  
![image](https://github.com/gin-gonic/website/assets/48476293/11dc094e-0848-4255-b61b-35226b640975)

**Comment by appleboy** [comment]
See the new website: https://github.com/gin-gonic/website/pull/289
